### PR TITLE
Configuring for pure-python-without-pypy

### PR DESCRIPTION
**Pull request by sallner**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+.coverage
 .installed.cfg
+.mr.developer.cfg
+.tox/
 bin/
-develop-eggs/
-src/zc.beforestorage.egg-info/
-parts/
-*.py[co]
-__pycache__/
```
Comment (mgedmin) [inline .gitignore:7]: Huh, the template doesn't ignore `__pycache__/`?  Well, I guess everything inside will be caught by the `*.pyc`, `*.pyo` rules.

File: tox.ini [hunk 2]
```diff
@@ -1,10 +1,49 @@
...
+
+[testenv:coverage]
+basepython = python3
+deps =
+    coverage
+    zope.testrunner
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report
```
Comment (mgedmin) [inline tox.ini:38]: I would prefer `coverage report -m`, but I guess I should address these comments to the zopefoundation/meta repository that has the template?

**Comment by mgedmin** [comment]
flake8 is unhappy with the current state of the codebase:

```
src/zc/beforestorage/tests.py:14:12: E401 multiple imports on one line
src/zc/beforestorage/tests.py:23:5: E306 expected 1 blank line before a nested definition, found 0
src/zc/beforestorage/tests.py:30:15: E203 whitespace before ':'
src/zc/beforestorage/tests.py:45:1: E305 expected 2 blank lines after class or function definition, found 1
src/zc/beforestorage/__init__.py:23:1: F401 'persistent' imported but unused
src/zc/beforestorage/__init__.py:26:1: E302 expected 2 blank lines, found 1
src/zc/beforestorage/__init__.py:29:57: E228 missing whitespace around modulo operator
src/zc/beforestorage/__init__.py:32:1: E302 expected 2 blank lines, found 1
src/zc/beforestorage/__init__.py:35:1: E305 expected 2 blank lines after class or function definition, found 1
src/zc/beforestorage/__init__.py:36:1: E302 expected 2 blank lines, found 0
src/zc/beforestorage/__init__.py:69:5: E303 too many blank lines (2)
src/zc/beforestorage/__init__.py:99:17: E125 continuation line with same indent as next logical line
src/zc/beforestorage/__init__.py:166:43: E251 unexpected spaces around keyword / parameter equals
src/zc/beforestorage/__init__.py:166:45: E251 unexpected spaces around keyword / parameter equals
src/zc/beforestorage/__init__.py:175:1: E302 expected 2 blank lines, found 1
src/zc/beforestorage/__init__.py:200:80: E501 line too long (84 > 79 characters)
setup.py:16:1: E402 module level import not at top of file
setup.py:17:1: E402 module level import not at top of file
```

I suggest adding an `allow_failures:` to `.travis.yml` for now, and then maybe cleaning it up in a separate PR?

**Comment by sallner** [comment]
I forgot to hit the draft button, Flake8 should run now.

File: CHANGES.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
+CHANGES
+=======
+
+0.6 (unreleased)
+----------------
+
+- Add support for Python 3.5 till 3.8.
```
Comment (dataflake) [inline CHANGES.rst:7]: Use "through" - "till" is only ever used in conjunction with time.

Comment (jamadden) [reply CHANGES.rst]: +1, "till" is the informal version of "until." 

The word "till" can also be a noun meaning "a cash register" or a verb meaning "prepare and cultivate land for crops."

Comment (dataflake) [reply CHANGES.rst]: Yes, but in this context...

File: .gitignore (hunk 1 above)
Comment (icemac) [reply .gitignore:7]: Fixed in zopefoundation/meta.

File: tox.ini (hunk 2 above)
Comment (icemac) [reply tox.ini]: Fixed in zopefoundation/meta.

We are currently working on the template trying to migrate some packages to gain experience and to find the best and agreed practices.

**Pull request merged**